website/docs/d/recovery_services_protection_policy_vm.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_recovery_services_protection_policy_vm"
+sidebar_current: "docs-azurerm-datasource-recovery-services-protection_policy_vm"
```

Comment:
a couple of things here - can we make this:

```suggestion
sidebar_current: "docs-azurerm-datasource-recovery-services-protection-policy-vm"
```

and then could we add a link to this [to the sidebar](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L242-L251)?